src/applications/caregivers/config/form.js

```diff
@@ -119,11 +120,19 @@ const formConfig = {
           uiSchema: vetContactInfoPage.uiSchema,
           schema: vetContactInfoPage.schema,
         },
-        veteranInfoThree: {
-          path: 'vet-3',
+        veteranInfoThreeJSON: {
+          path: 'vet-3-json',
           title: 'VA medical center',
-          uiSchema: vetMedicalCenterPage.uiSchema,
-          schema: vetMedicalCenterPage.schema,
+          depends: formData => !formData['view:useFacilitiesAPI'],
+          uiSchema: vetMedicalCenterJsonPage.uiSchema,
+          schema: vetMedicalCenterJsonPage.schema,
```

Comment:
Without seeing these pages, would a Veteran know what to do with JSON data?

Also, the path names could maybe be improved? I'm not an IA person, but in the past, I've named the path similar to the page title.

Reply:
This is only temporary, as we will be doing a phased in approach. Once the phase in is complete and we're satisfied, all the naming conventions go back to as they are now and the current codebase that fetches the static JSON file will go away.